lib/aerosol/runner.rb

```diff
@@ -72,8 +72,13 @@ def wait_for_new_instances
         info "waiting for instances to be live (#{remaining_instances.count} remaining)"
         debug "current instances: #{current_instances.map(&:id)}"
         debug "live instances: #{live_instances.map(&:id)}"
-        live_instances.concat(remaining_instances.select { |instance| healthy?(instance) })
+        new_live_instances = remaining_instances.select { |instance|
+          debug "Checking if #{instance.id} is healthy"
```

Comment:
You could leave this block alone if you moved the debug log to the first statement in `healthy?`